nilearn/interfaces/fmriprep/tests/test_load_confounds.py

```diff
@@ -391,7 +391,8 @@ def test_not_found_exception(tmp_path):
     # Aggressive ICA-AROMA strategy requires
     # default nifti
     aroma_nii, _ = create_tmp_filepath(
-        tmp_path, image_type="ica_aroma", suffix="aroma"
+        tmp_path, image_type="ica_aroma", bids_fields={"entities":
+                                                       {"sub": "aro-ma"}}
```

Comment:
This is not a valid bids participant ID pattern. Why are we using this as a test case? Is it a problem that it's not breaking things?

Reply:
Ah. Need to double check. I had to make a change there during testing things. Let me see.

Reply:
OK my bad this change was not required: I must have misunderstood the reason why this test failed at one point during refactoring